The failure on the lint/build doesn't look like something I can fix?

$ make lint
poetry run mypy .
venv\Scripts\activate_this.py:27: error: "str" has no attribute "decode"; maybe "encode"?  [attr-defined]
venv\Scripts\activate_this.py:30: error: Module has no attribute "real_prefix"  [attr-defined]
Found 2 errors in 1 file (checked 717 source files)
make: *** [lint] Error 1

--- a/docs/modules/indexes/document_loaders/examples/word_document.ipynb
+++ b/docs/modules/indexes/document_loaders/examples/word_document.ipynb
@@ -11,11 +11,81 @@
    ]
   },
   {
+   "cell_type": "markdown",
+   "id": "9438686b",
+   "metadata": {},
+   "source": [
+    "## Using Docx2txt\n",
+    "\n",
+    "Load .docx using `Docx2txt` into a document."
+   ]
+  },
+  {
    "cell_type": "code",
-   "execution_count": 1,
-   "id": "721c48aa",
+   "execution_count": null,
+   "id": "7b80ea89",
    "metadata": {},
    "outputs": [],
+   "source": [
+    "from langchain.document_loaders import Docx2txtLoader"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "99a12031",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "loader = Doc2txtLoader(\"example_data/fake.docx\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b92f68b0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = loader.load()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d83dd755",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8d40727d",
+   "metadata": {},
+   "source": [
+    "## Using Unstructured"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "721c48aa",
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "ModuleNotFoundError",
+     "evalue": "No module named 'langchain'",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[1;31mModuleNotFoundError\u001b[0m                       Traceback (most recent call last)",
+      "Cell \u001b[1;32mIn[3], line 1\u001b[0m\n\u001b[1;32m----> 1\u001b[0m \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01mlangchain\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01mdocument_loaders\u001b[39;00m \u001b[38;5;28;01mimport\u001b[39;00m UnstructuredWordDocumentLoader\n",
+      "\u001b[1;31mModuleNotFoundError\u001b[0m: No module named 'langchain'"
+     ]
+    }
+   ],
    "source": [
     "from langchain.document_loaders import UnstructuredWordDocumentLoader"
    ]
@@ -129,7 +199,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.13"
+   "version": "3.10.10"
   }
  },
  "nbformat": 4,

--- a/langchain/document_loaders/__init__.py
+++ b/langchain/document_loaders/__init__.py
@@ -78,7 +78,7 @@ from langchain.document_loaders.url_playwright import PlaywrightURLLoader
 from langchain.document_loaders.url_selenium import SeleniumURLLoader
 from langchain.document_loaders.web_base import WebBaseLoader
 from langchain.document_loaders.whatsapp_chat import WhatsAppChatLoader
-from langchain.document_loaders.word_document import UnstructuredWordDocumentLoader
+from langchain.document_loaders.word_document import UnstructuredWordDocumentLoader, Docx2txtLoader
 from langchain.document_loaders.youtube import (
     GoogleApiClient,
     GoogleApiYoutubeLoader,
@@ -162,4 +162,5 @@ __all__ = [
     "PythonLoader",
     "ChatGPTLoader",
     "HuggingFaceDatasetLoader",
+    "Docx2txtLoader",
 ]

